src/partials/partial-sky-js.hbs

```diff
@@ -1 +1 @@
-<script src="https://sky.blackbaudcdn.net/skyux/{{ stache.config.latest_sky_release }}/js/sky-bundle.min.js"></script>
+<script src="//sky.blackbaudcdn.net/skyux/{{ stache.config.latest_sky_release }}/js/sky-bundle.min.js"></script>
```

Comment:
We force https to support the following scenario.  User runs `grunt build` and then uses the `file://` protocol to open local `index.html`, which attempts to load SKY `file://sky.blackbaudcdn.net...` which will obviously fail.

Super old versions of IE used to fail when loading secure content over an insecure connection, but that's no longer the case with browsers that SKY UX (and Stache) support.